src/lua/fiber_cond.h

```diff
@@ -30,6 +30,7 @@
  * THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
  * SUCH DAMAGE.
  */
+#include <fiber_cond.h>
```

Comment:
Please don't include headers unless absolutely necessary. You can use a forward declaration instead:

```C
struct fiber_cond;
```

Reply:
fixed